pom.xml

```diff
@@ -355,11 +374,11 @@
             <hudson.agents.NodeProvisioner.MARGIN0>1.0</hudson.agents.NodeProvisioner.MARGIN0>
             <hudson.remoting.Launcher.pingIntervalSec>-1</hudson.remoting.Launcher.pingIntervalSec>
             <com.google.jenkins.plugins.computeengine.integration.ITUtil.windows>${it.windows}</com.google.jenkins.plugins.computeengine.integration.ITUtil.windows>
+            <jenkins.test.timeout>600</jenkins.test.timeout>
```

Comment:
Note: normally would use `@WithTimeout` on specific tests expected to be slow. Apparently this only supports `@Rule JenkinsRule` and not `@ClassRule`.